### ISSUE TITLE
Solr_Configure exits with 0 on failure

### ISSUE THREAD

**Issue by micmania1** [silverstripe/silverstripe-fulltextsearch#146]
Solr_Configure failed as it didn't have permission to create the `.solr` directory but its exit code was still 0. Its not easy to detect failure in deployments which rely on it or in CI.

```
sudo -u www-data php framework/cli-script.php dev/tasks/Solr_Configure
Running Task Solr_Configure

[2017-06-22 10:55:47] solr_configure.ERROR: Failure: Failed creating target directory /home/ubuntu/{project}/.solr/SolrSearchIndex/conf, please check permissions [] []
[2017-06-22 10:55:47] solr_configure.ERROR: Failure: Failed creating target directory /home/ubuntu/{project}/.solr/FAQSearchIndex/conf, please check permissions [] []
```

**Comment by robbieaverill** [comment]
@micmania1 patch at #147 - seem reasonable?

**Comment by robbieaverill** [comment]
Fixed in #147
